prov/efa/src/rdm/rxr_ep.c

```diff
@@ -1410,25 +1401,36 @@ int rxr_ep_init(struct rxr_ep *ep)
 	return ret;
 }
 
-static int rxr_ep_rdm_setname(fid_t fid, void *addr, size_t addrlen)
+static int rxr_ep_rdm_getname(fid_t fid, void *addr, size_t *addrlen)
```

Comment:
This is a copy of `efa_ep_getname` from `dgram/efa_ep.c`. I think we should move that function to efa_base_ep, and let both dgram_ep and rdm use it.

Reply:
thanks for the catch - fixed